File: app/src/main/java/tr/com/cherrysunshinysky/todonotesapp/DetailActivity.java
package tr.com.cherrysunshinysky.todonotesapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

import static tr.com.cherrysunshinysky.todonotesapp.PrefConstant.DESCRIPTION;
import static tr.com.cherrysunshinysky.todonotesapp.PrefConstant.TITLE;

public class DetailActivity extends AppCompatActivity {

    TextView tvDetailTitle, tvDetailDescription;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindViews();
        getDataFromMyNotesActivity();
    }

    private void getDataFromMyNotesActivity() {
        Intent intent = getIntent();
        String title = intent.getStringExtra(TITLE);
        String description = intent.getStringExtra(DESCRIPTION);

        tvDetailTitle.setText(title);
        tvDetailDescription.setText(description);
    }

    private void bindViews() {
        tvDetailTitle = findViewById(R.id.tv_detail_title);
        tvDetailDescription = findViewById(R.id.tv_detatil_description);
    }
}

File: app/src/main/java/tr/com/cherrysunshinysky/todonotesapp/DetailActivity.kt
package tr.com.cherrysunshinysky.todonotesapp

import android.content.Intent
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

/**
 * Created by Emir U. Özen on 3/12/21.
 * emir.ozen@outlook.com
 */
class DetailActivity : AppCompatActivity() {
    private lateinit var tvDetailTitle: TextView
    private lateinit var tvDetailDescription: TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindViews()
        getDataFromMyNotesActivity()
    }

    private fun getDataFromMyNotesActivity() {
        val intent = intent
        val title: String? = intent.getStringExtra(PrefConstant.TITLE)
        val description: String? = intent.getStringExtra(PrefConstant.DESCRIPTION)

        tvDetailTitle.text = title
        tvDetailDescription.text = description
    }

    private fun bindViews() {
        tvDetailTitle = findViewById(R.id.tv_detail_title)
        tvDetailDescription = findViewById(R.id.tv_detatil_description)
    }
}